### DISCUSSION TITLE
Unable to quote command line properly

### DISCUSSION THREAD

**Discussion by johnfredcee** [microsoft/vscode-cpptools#8323] [Q&A]
```I am trying to create a command line option -LogCmds=\"LogGDB all\" on the command line, but am unable to form a correctly formed string in the workspace - the closest I can get to it is  "-LogCmds=\\\\\"LogGDB all\\\\\"" which resolves to -LogCmds="\"LogGDB all\"" ```

I don't see where the extra quotes come from, but the quoting rules are not fully documented, so I'm not sure if this is a bug.
This is with 1.62.2 (user setup) and v 1.7.1 of the extension.

**Comment by sean-mcmanus** [comment]
What is the command line option and what file is it in, i.e. what program is supposed to process it?
